src/scenes/Projects/Overview/ProjectOverview.graphql

```diff
@@ -24,7 +24,28 @@ fragment ProjectOverview on Project {
       ...DisplayLocation
     }
   }
-  marketingLocation {
+  marketingCountry {
+    canRead
+    canEdit
+    value {
+      ...DisplayLocation
+    }
+  }
```

Comment:
Since we're reusing this in so many places, I'd probably just make a fragment and reuse it.